firestore/FirestoreSwiftUIExample/Views/RestaurantItemView.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  RestaurantView.swift
+//  FirestoreSwiftUIExample
+//
+
+import SwiftUI
+import SDWebImageSwiftUI
+
+struct RestaurantItemView: View {
+    var restaurant: Restaurant
+
+    var body: some View {
+        HStack {
+            ImageThumbnail(imageURL: restaurant.photo)
+            VStack(alignment: .leading) {
+                HStack {
+                    Text(restaurant.name)
+                        .frame(alignment: .leading)
+                    Spacer()
+                    Price(price: restaurant.price)
+                }
+                StarsView(rating: Int(restaurant.averageRating.rounded()))
+                Spacer()
+                HStack {
+                    Text(restaurant.category)
+                    Text("•")
+                    Text(restaurant.city)
+                }
+                    .foregroundColor(Color.gray)
+                    .font(.footnote)
+            }
+        }
+        .padding([.bottom, .trailing])
+    }
+}
+
+struct RestaurantItemView_Previews: PreviewProvider {
+    static var previews: some View {
+        let data = Restaurant(id: .init(), name: "Pizza Place", category: "Pizza", city: "Austin", price: 2, ratingCount: 1, averageRating: 4, photo: Restaurant.imageURL(forName: "Place Place"))
+        RestaurantItemView(restaurant: data)
+    }
+}
+
+struct ImageThumbnail: View {
+    var imageURL: URL
+
+    var body: some View {
+        WebImage(url: imageURL)
+            .resizable()
+            .placeholder(Image(systemName: "photo"))
+            .aspectRatio(1, contentMode: .fill)
+            .frame(width: 100, height: 100, alignment: .leading)
+    }
+}
+
+struct Price: View {
```

Comment:
This might go into a separate file (esp. if it's reused elsewhere).

Might also want to call it `PriceView`